devices.js

```diff
@@ -8904,6 +8904,159 @@ const devices = [
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint1 = device.getEndpoint(1);
+            await bind(endpoint1, coordinatorEndpoint, ['genOnOff']);
+            const endpoint2 = device.getEndpoint(2);
+            await bind(endpoint2, coordinatorEndpoint, ['genOnOff']);
+            const endpoint3 = device.getEndpoint(3);
+            await bind(endpoint3, coordinatorEndpoint, ['genOnOff']);
+        },
+        endpoint: (device) => {
+            return {'top': 1, 'center': 2, 'bottom': 3};
+        },
+    },
+    {
+        zigbeeModel: ['074b3ffba5a045b7afd94c47079dd553'],
+        model: 'T21W2Z',
+        vendor: 'ORVIBO',
+        description: 'Smart light switch - 2 gang',
+        supports: 'on/off',
+        fromZigbee: [fz.on_off, fz.ignore_basic_report],
+        toZigbee: [tz.on_off],
+        endpoint: (device) => {
+            return {'top': 1, 'bottom': 2};
+        },
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint) => {
+            await bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
+            await bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
+        },
+    },
+    {
+        zigbeeModel: ['095db3379e414477ba6c2f7e0c6aa026'],
+        model: 'T21W1Z',
+        vendor: 'ORVIBO',
+        description: 'Smart light switch - 1 gang',
+        supports: 'on/off',
+        fromZigbee: [fz.on_off],
+        toZigbee: [tz.on_off],
+    },
+    {
+        zigbeeModel: ['093199ff04984948b4c78167c8e7f47e'],
+        model: 'W40CZ',
+        vendor: 'Orvibo',
+        description: 'Orvibo smart curtain motor ',
+        supports: 'open, close, stop, position',
+        fromZigbee: [fz.ZNCLDJ11LM_ZNCLDJ12LM_curtain_analog_output, fz.cover_position_tilt, fz.ignore_basic_report],
+        toZigbee: [tz.cover_state, tz.cover_position_tilt],
+        onEvent: async (type, data, device) => {
+            if (data.type === 'attributeReport' && data.cluster === 'genAnalogOutput') {
+                await device.endpoints[0].read('genAnalogOutput', ['presentValue']);
+            }
+        },
+    },
+    {
+        zigbeeModel: ['e0fc98cc88df4857847dc4ae73d80b9e'],
+        model: 'R11W2Z',
+        vendor: 'ORVIBO',
+        description: 'In wall switch - 2 gang',
+        supports: 'on/off',
+        fromZigbee: [fz.on_off],
+        toZigbee: [tz.on_off],
+        endpoint: (device) => {
+            return {'l1': 1, 'l2': 2};
+        },
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint) => {
+            await bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
+            await bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
+        },
+    },
+    {
+        zigbeeModel: ['9ea4d5d8778d4f7089ac06a3969e784b'],
+        model: 'R20W2Z',
+        vendor: 'ORVIBO',
+        description: 'In wall switch single Live',
```

Comment:
Why is this not named `In wall switch - 1 gang`? (to be consistent with the `R11W2Z`)

Reply:
This is what was written on the description of the device, and actually it's not 1gang but 2gang switch. I think would be better to change to `In wall switch - 2 gang`